README.md

```diff
@@ -76,6 +85,13 @@ and the following Typescript types:
 - `GoResult<T> = { data: T; success: true }`
 - `GoResultSuccess<E extends Error = Error> = { error: E; success: false }`
 - `GoResultError<T, E extends Error = Error> = GoResultSuccess<T> | GoResultError<E>`
+- `PromiseOptions { readonly retries?: number; readonly retryDelayMs?: number; readonly timeoutMs?: number; }`
```

Comment:
Nit: I wonder about the PromiseOptions name, but I do not have a better idea...

Reply:
`RetryOptions`? :smile:

Reply:
But it also does timeout so retry options is not accurate. Maybe just `Options` but I don't mind this way too.